VirtoCommerce.AzureSearchModule.Tests/packages.config

```diff
@@ -1,9 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Castle.Core" version="4.1.0" targetFramework="net461" />
-  <package id="Moq" version="4.7.63" targetFramework="net461" />
+  <package id="Castle.Core" version="4.1.1" targetFramework="net461" />
+  <package id="Moq" version="4.7.99" targetFramework="net461" />
   <package id="Newtonsoft.Json" version="11.0.2" targetFramework="net461" />
-  <package id="VirtoCommerce.CoreModule.Search.Tests" version="2.23.1" targetFramework="net461" />
+  <package id="VirtoCommerce.CoreModule.Search.Tests" version="2.24.21" targetFramework="net461" />
```

Comment:
This should be 2.25.16 because this module depends on VirtoCommerce.Core 2.25.16

Reply:
ok